lib/global.js

```diff
@@ -1,14 +1,3 @@
-var createTorrent = require('create-torrent')
-var wrtc = require('wrtc')
+var wrtc = require('electron-webrtc')()
 
 global.WRTC = wrtc
-
-global.WEBTORRENT_ANNOUNCE = createTorrent.announceList
```

Comment:
Thought the default trackers will be added to the torrents anyway (see `announceList`). Is there a reason why websocket trackers should be forced only at standalone versions like the app/hybrid and not the browser ?